app/build.gradle

```diff
@@ -319,6 +319,10 @@ dependencies {
 
     // Desugaring Java 8 to < API 26
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.0.3'
+
+    // Coroutines
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4'
```

Comment:
Hey that's not only what to do to add the coroutine on a project please have a look at the documentation https://github.com/Kotlin/kotlinx.coroutines/blob/master/README.md#android